.github/workflows/release.yml

```diff
@@ -1,8 +1,10 @@
 name: Release
 on:
-  push:
-    branches:
-      - master
+  workflow_run:
+    workflows: [Lint, 'Unit tests']
+    branches: [master]
+    types:
+      - completed
```

Comment:
I _assume_ this is "completed successfully". Might be wrong though

Reply:
When you assume...

https://github.community/t/workflow-run-success-type-reference/133194/2

Reply:
Oh yeah no this doesn't run only on success - I played around with it last month for something else and found this out 😬